src/config.ts

```diff
@@ -619,6 +619,6 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 			},
 		],
 		features: ['stargate', 'ibc-transfer'],
-		explorerUrlToTx: 'http://junoscan.com/tx/${txHash}',
+		explorerUrlToTx: 'https://junoscan.com/transactions/{txHash}',
```

Comment:
This should need the `$`, no?

Reply:
nope 
https://github.com/osmosis-labs/osmosis-frontend/blob/87494115a41ca3391b2d6b59559275c575f42a94/src/config.ts#L246

Reply:
@ValarDragon Juno is available on mintscan, will it be used?

Reply:
We can use that instead, I don't really have an opinion on junoscan vs mintscan, other than mintscan is convenient since its the same as most other chains